test/integration/emitter_singleton_test.ts

```diff
@@ -0,0 +1,56 @@
+import "mocha";
+
+import { emitterFor, HTTP, Mode, Message, Emitter } from "../../src";
+
+import { fixture, assertStructured } from "./emitter_factory_test";
+
+import { rejects, doesNotReject } from "assert";
+
+describe("Emitter Singleton", () => {
+  it("emit", async () => {
```

Comment:
Maybe some more descriptive text for the test case would be good.

```suggestion
  it("emits a Node.js 'cloudevent' event as an EventEmitter", async () => {
```

Reply:
It is part of the wider group `EventEmitter`